beacon-chain/chaintest/backend/simulated_backend_test.go

```diff
@@ -27,7 +27,7 @@ func TestRunShuffleTest(t *testing.T) {
 	}
 	testCase := &ShuffleTestCase{
 		Input:  []uint32{1, 2, 3, 4, 5},
-		Output: []uint32{2, 4, 1, 3, 5},
+		Output: []uint32{2, 4, 5, 1, 3},
```

Comment:
why was this changed ?

Reply:
Shuffle test uses the hashing function. Diff hashing algorithm produces different shuffled output...